Sources/StreamChat/Repositories/MessageRepository.swift

```diff
@@ -56,8 +62,8 @@ class MessageRepository {
             }, completion: { error in
                 if let error = error {
                     log.error("Error changing localMessageState message with id \(messageId) to `sending`: \(error)")
-                    self?.markMessageAsFailedToSend(id: messageId) {
-                        completion(.failure(.failedToSendMessage))
+                    self?.markMessageAsFailedToSend(id: messageId, isForced: true) { error in
```

Comment:
There was an obvious bug where `markMessageAsFailedToSend` doesn't mark the message's status to `.sendingFailed`.

`isForced: true` fixes that.